snapcraft/cmds.py

```diff
@@ -229,7 +229,7 @@ def run(args):
             ['ssh'] + ssh_opts +
             ['ubuntu@localhost', '-p', '8022', 'sudo snappy install  *.snap'])
         # "login"
-        _check_call(
+        _call(
```

Comment:
It's CalledProcessError, and if you prefer I can just catch that one. It's also very easy to reproduce by just doing:
snapcraft run
(in session) false
exit

Here's the traceback I get:
Connection to localhost closed.
Traceback (most recent call last):
  File "/usr/bin/snapcraft", line 33, in <module>
    snapcraft.main.main()
  File "/usr/lib/python3/dist-packages/snapcraft/main.py", line 147, in main
    args.func(args)
  File "/usr/lib/python3/dist-packages/snapcraft/cmds.py", line 234, in run
    preexec_fn=os.setsid)
  File "/usr/lib/python3/dist-packages/snapcraft/cmds.py", line 362, in _check_call
    return subprocess.check_call(args, **kwargs)
  File "/usr/lib/python3.4/subprocess.py", line 561, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['ssh', '-i', '/home/plars/.ssh/id_rsa', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/tmp/tmpdteru9rp', '-oKbdInteractiveAuthentication=no', '-p', '8022', 'ubuntu@localhost']' returned non-zero exit status 130

I think it's safe to just use _call here, but if you'd prefer to specifically catch CalledProcessError, I'll be happy to change it.